sip-core/src/main/java/de/ikor/sip/foundation/core/declarative/connector/GenericInboundConnectorBase.java

```diff
@@ -28,8 +29,10 @@ public final void defineInboundEndpoints(
       final RoutesDefinition definition,
       final EndpointProducerBuilder targetToDefinition,
       final RoutesRegistry routeRegistry) {
+    EndpointConsumerBuilder endpointConsumerBuilder =
+        resolveEndpointProperties(defineInitiatingEndpoint());
     definition
-        .from(defineInitiatingEndpoint())
+        .from(endpointConsumerBuilder)
```

Comment:
Are we covered on the Rest Connector also? There is some difference in implementation there.